### PR TITLE
Attempt to improve documentation for src/emcpy/calcs

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Adds __all__ as well as some detailed docstrings.

### CODE REVIEW

File: src/emcpy/calcs/units.py [hunk 1]
```diff
@@ -1,36 +1,102 @@
+__all__ = ['K_to_C', 'K_to_F', 'C_to_K', 'C_to_F', 'F_to_C', 'mps_to_MPH',
+           'mm_to_inches']
```
Comment (aerorahul) [inline src/emcpy/calcs/units.py]: do we not ever need `inches_to_mm`?

File: src/emcpy/calcs/units.py [hunk 2]
```diff
@@ -1,36 +1,102 @@
...
-    """Convert Kelvin to Celsius"""
+    """
+    K_to_C : Convert Kelvin to Celsius
+
+    K_to_C(K)
+
+    INPUT:
+      K - temperature in kelvin
+    OUTPUT:
+      C - temperature in degrees celsius
```
Comment (aerorahul) [inline src/emcpy/calcs/units.py]: Ok. So we need to converge on a nomenclature.
See `io.netCDF` docstrings.

I am fine with whichever way we go, we just have to be consistent.

File: src/emcpy/calcs/wind.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import numpy as np
 
+__all__ = ['uv_to_speed', 'spddir_to_uv']
```
Comment (aerorahul) [inline src/emcpy/calcs/wind.py]: I suggest we change `uv_to_spddir` to be consistent with the adjoint operation.

**Review by aerorahul — COMMENTED** [review]
minor comments.
ISO, opinion on converging to a singular docstring format.

File: src/emcpy/calcs/units.py (hunk 1 above)
Comment (CoryMartin-NOAA) [reply src/emcpy/calcs/units.py]: yes there should probably be MPH to mps also.

File: src/emcpy/calcs/units.py (hunk 2 above)
Comment (CoryMartin-NOAA) [reply src/emcpy/calcs/units.py]: Ok I don't have a strong opinion on which way to go either, as the `stats.py` file you added has different templates for it. @kevindougherty-noaa any preference on a way to go?

File: src/emcpy/calcs/wind.py (hunk 3 above)
Comment (CoryMartin-NOAA) [reply src/emcpy/calcs/wind.py]: agreed, that makes sense.

File: src/emcpy/calcs/units.py (hunk 2 above)
Comment (kevindougherty-noaa) [reply src/emcpy/calcs/units.py]: Based on my research on Friday, I think the numpy/scipy docstrings is the best way to go. It is a combination of reStructured text which is 'feature rich' and Google's docstring which I think is cleaner. See attached link for info on each: https://realpython.com/documenting-python-code/#docstring-formats

Comment (CoryMartin-NOAA) [reply src/emcpy/calcs/units.py]: Works for me, I also think INPUT and OUTPUT scream FORTRAN, so I think this is a good choice.

Comment (aerorahul) [reply src/emcpy/calcs/units.py]: works for me too.

**Review by aerorahul — APPROVED** [review]
looks good.

**Pull request merged**